include/crc7.h

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
need to move this to the unified crc.h include file

Reply:
Done in b2375eeda10fef322fec20a68189bdc35ebd2539